ciphers/vigenere_cipher.py

```diff
@@ -1,11 +1,25 @@
+import secrets
+
 LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
 
 def main() -> None:
     message = input("Enter message: ")
-    key = input("Enter key [alphanumeric]: ")
+
+    print(
+        "\nEnter key [alphanumeric], "
+        "leave blank for a random key as long as the message."
+    )
+    key = str(input("Enter the Key: ").strip())
```

Comment:
```suggestion
    key = input("Enter the Key: ").strip()
```
The `str` call is unnecessary because `input` and `strip` already return strings